hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestObjectStoreWithFSO.java

```diff
@@ -515,12 +517,11 @@ public void testListKeysWithNotNormalizedPath() throws Exception {
     byte[] input = new byte[length];
     Arrays.fill(input, (byte)96);
```

Comment:
`input` becomes unused by calling the method that creates its own input, so it can be removed.